src/Neo/Persistence/DataCache.cs

```diff
@@ -42,9 +42,20 @@ public class Trackable(StorageItem item, TrackState state)
             public TrackState State { get; set; } = state;
         }
 
+        /// <summary>
+        /// Delegate for storage entries
+        /// </summary>
+        /// <param name="sender">DataCache</param>
+        /// <param name="key">Key</param>
+        /// <param name="item">Item</param>
+        public delegate void DelOnEntry(DataCache sender, StorageKey key, StorageItem item);
```

Comment:
Delegate with short of `del` sort of misleading to `delete`

Reply:
`OnEntryDelegate`?